config/fund_loader_config/hsra/hsra.py

```diff
@@ -0,0 +1,172 @@
+        "tree_path": f"{APPLICATION_BASE_PATH_HSRA}.3.1",
+    },
+    {
+        "section_name": {"en": "Designated area details", "cy": ""},
+        "form_name_json": {"en": "designated-area-details-hsra", "cy": ""},
+        "tree_path": f"{APPLICATION_BASE_PATH_HSRA}.3.2",
+    },
+    {
+        "section_name": {"en": "Project milestones", "cy": ""},
+        "form_name_json": {"en": "milestones-hsra", "cy": ""},
+        "tree_path": f"{APPLICATION_BASE_PATH_HSRA}.3.3",
+    },
+    {
+        "section_name": {"en": "4. Project costs", "cy": ""},
+        "tree_path": f"{APPLICATION_BASE_PATH_HSRA}.4",
+    },
+    {
+        "section_name": {"en": "Total expected cost", "cy": ""},
+        "form_name_json": {"en": "total-expected-cost-hsra", "cy": ""},
+        "tree_path": f"{APPLICATION_BASE_PATH_HSRA}.4.1",
+    },
+    {
+        "section_name": {"en": "Refurbishment costs", "cy": ""},
+        "form_name_json": {"en": "refurbishment-costs-hsra", "cy": ""},
+        "tree_path": f"{APPLICATION_BASE_PATH_HSRA}.4.2",
+    },
+    {
+        "section_name": {"en": "Auction costs", "cy": ""},
+        "form_name_json": {"en": "auction-costs-hsra", "cy": ""},
+        "tree_path": f"{APPLICATION_BASE_PATH_HSRA}.4.3",
+    },
+    {
+        "section_name": {"en": "Other costs", "cy": ""},
+        "form_name_json": {"en": "other-costs-hsra", "cy": ""},
+        "tree_path": f"{APPLICATION_BASE_PATH_HSRA}.4.4",
+    },
+    {
+        "section_name": {"en": "5. Declaration", "cy": ""},
+        "tree_path": f"{APPLICATION_BASE_PATH_HSRA}.5",
+    },
+    {
+        "section_name": {"en": "Declaration", "cy": ""},
+        "form_name_json": {"en": "declaration-hsra", "cy": ""},
+        "tree_path": f"{APPLICATION_BASE_PATH_HSRA}.5.1",
+    },
+]
+
+fund_config = {
+    "id": HSRA_FUND_ID,
+    "name_json": {
+        "en": "High Street Rental Auctions Fund",
+        "cy": "",
+    },
+    "title_json": {
+        "en": "funding to cover the cost of delivering a high street rental auction",
+        "cy": "",
+    },
+    "short_name": "HSRA",
+    "description_json": {"en": "", "cy": ""},
+    "welsh_available": False,
+    "owner_organisation_name": "Department for Levelling Up, Housing and Communities",
+    "owner_organisation_shortname": "DLUHC",
+    "owner_organisation_logo_uri": DLUHC_LOGO_PNG,
+}
+
+round_config = [
+    {
+        "id": HSRA_ROUND_ID,
+        "fund_id": HSRA_FUND_ID,
+        "title_json": {"en": "", "cy": ""},
+        "short_name": "R1",
+        "opens": HSRA_OPENS_DATE,
+        "assessment_start": None,
+        "deadline": HSRA_DEADLINE_DATE,
+        "application_reminder_sent": False,
+        "reminder_date": HSRA_SEND_REMINDER_DATE,
+        "assessment_deadline": HSRA_ASSESSMENT_DEADLINE_DATE,
+        "prospectus": "",  # TODO needs to be added
+        "privacy_notice": "",
+        "contact_email": "HighStreetRentalAuctions@levellingup.gov.uk",
```

Comment:
Could we maybe add the new field: 

```suggestion
        "contact_us_banner_json": {"en": "", "cy": ""},
        "contact_email": "HighStreetRentalAuctions@levellingup.gov.uk",
```

So we don't see failure when it get merged and loaded? This work was ongoing at same time so makes sense how it's not here just now.